src/App.scss

```diff
@@ -40,10 +40,10 @@ body {
 .main {
   flex: 1 1 auto;
   overflow: hidden;
-  padding: 140px;
```

Comment:
You are removing bottom padding too with this changes, and the footer overlaps the main content

Reply:
fixed it by adding padding-bottom